hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxySupport.java

```diff
@@ -1047,5 +1048,9 @@ public void onFailure(Throwable t) {
     public void setOperationProvider(MapOperationProvider operationProvider) {
         this.operationProvider = operationProvider;
     }
+
+    public int getBackupCount() {
+        return mapConfig.getBackupCount();
```

Comment:
It follows the same implementation as `MapContainer.getBackupCount` which only takes into account `mapConfig.getBackupCount`.